client/src/pages/Home/index.js

```diff
@@ -1,24 +1,65 @@
-import { useEffect, useState } from "react";
+import React, { useEffect, useState } from "react";
 
 import { Pagination, ResourceList } from "../../components";
 import { useSearchParams } from "../../hooks";
 import { ResourceService, useService } from "../../services";
 
-export function Home() {
+const Home = () => {
```

Comment:
❓ Why switch from vanilla to arrow function here? I don't think any of the other components in this project are arrow functions.